src/client/datascience/notebookExtensibility.ts

```diff
@@ -1,8 +1,4 @@
-import { injectable } from 'inversify';
-import { Event, EventEmitter, NotebookCell, Uri } from 'vscode';
-import { noop } from '../common/utils/misc';
-import { ICell, INotebookExecutionLogger, INotebookExtensibility } from './types';
-import { translateCellToNative } from './utils';
+import { NotebookCell, Uri } from 'vscode';
```

Comment:
Maybe rename this file? Or just move types somewhere else?

Reply:
I'd like to leave that for debt, i just want to get this PR over and done with and get it into the release branch.
Hope thats ok with you.